rest/rest-resources/src/main/resources/OSGI-INF/blueprint/service.xml

```diff
@@ -34,71 +36,34 @@ limitations under the License.
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="entityConfigResourceBean" class="org.apache.brooklyn.rest.resources.EntityConfigResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="entityResourceBean" class="org.apache.brooklyn.rest.resources.EntityResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="locationResourceBean" class="org.apache.brooklyn.rest.resources.LocationResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="policyConfigResourceBean" class="org.apache.brooklyn.rest.resources.PolicyConfigResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="policyResourceBean" class="org.apache.brooklyn.rest.resources.PolicyResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="scriptResourceBean" class="org.apache.brooklyn.rest.resources.ScriptResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="sensorResourceBean" class="org.apache.brooklyn.rest.resources.SensorResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="serverResourceBean" class="org.apache.brooklyn.rest.resources.ServerResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="usageResourceBean" class="org.apache.brooklyn.rest.resources.UsageResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
-    <bean id="versionResourceBean" class="org.apache.brooklyn.rest.resources.VersionResource">
-        <property name="managementContext" ref="localManagementContext" />
-        <property name="managementContextInternal" ref="localManagementContextInternal" />
-    </bean>
+    <bean id="applicationResourceBean" class="org.apache.brooklyn.rest.resources.ApplicationResource" />
```

Comment:
must remember -- or document -- that this file needs updating if we add rest resources.

nice clean up.

feels like we should brain dump to the dev guide a page on how rest and ui code all works.